x-pack/packages/lens/icons/src/assets/index.scss

```diff
@@ -0,0 +1,15 @@
+.lensChartIcon__subdued {
+  fill: $euiTextSubduedColor;
+
+  .lnsLayerChartSwitch__item-isSelected & {
+    fill: currentColor;
+  }
```

Comment:
Is this `lnsLayerChartSwitch__item-isSelected` class actually in use? I looked but couldn't find anywhere that was using it. If not in use anywhere, I imagine we should be able to delete this.

Reply:
you are probably right, I also cannot find any places where we use it. Also if somewhere we still use that, not sure that it's a good place for that. Let's remove